src/content/Home/main1.md

```diff
@@ -5,14 +5,13 @@
 
 # Our Mission
 
-This is some text about how great HKN. I am building the website for HKN. This text should be relatively long, but not too long. I still need to refactor the code for this website and add some more new features. This is some more text, HKN does fun events like happy hour. Even more text, and even more text, and even more text! This is a long paragraph, but is necessary to make things look balanced, it's height should be not much less than that of the image below.
+Eta Kappa Nu (HKN) strives to assist its members and those in the broader Illinois ECE community throughout their lives in becoming better professionals by encouraging its members to improve the standards of their profession, courses of instruction, and the institutions where they belong. In this spirit, we provide an array of services to Illinois students, including free one-on-one peer tutoring and exam review sessions for core ECE courses.
+
+HKN membership is selective. Prospective members, known as initiates, must not only be in the top third of their class by GPA, but must also complete at least 10 hours of service in a semester to demonstrate their character and attitude towards the community.
```

Comment:
"must not only be in the top third of their class by GPA, but must also complete" -> "must be in the top third of their class by GPA, and complete at least 10 hours of service"

Maybe write 10 as ten?